tests/hooks/test_handoff_stop.py

```
@@ -481,40 +495,64 @@ def test_f14_subagent_stop_event_silent(tmp_path: Path) -> None:
+        "hookSpecificOutput must not appear in Stop-hook output"
+    )
+
+
+def test_f15_enforce_uses_continue_false_and_decision_block(tmp_path: Path) -> None:
     handoff = _make_handoff(require_review=True)
     result = _run_gate(_stop_event(), mode="enforce", handoff=handoff, tmp_path=tmp_path)
     assert result is not None
-    assert result.get("hookEventName") == "Stop"
+    assert result.get("continue") is False
+    assert result.get("decision") == "block"
+    assert isinstance(result.get("reason"), str) and result["reason"]
```

Ruff (0.15.14)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)